galaxy/wrapper/hicBuildMatrix.xml

```diff
@@ -119,7 +119,7 @@
             </repeat>
             <param name="binSize" value="5000"/>
             <output name="outBam" file="hicBuildMatrix_result1.bam" ftype="bam"/>
-            <output name="outFileName" file="hicBuildMatrix_result1.h5" ftype="h5" compare="sim_size"/>
+            <output name="outFileName" file="hicBuildMatrix_result1.h5" ftype="h5" compare="sim_size" delta="30000"/>
```

Comment:
To quote myself from PR #89:
'I increased the delta for the h5 matrix of the test case to 30000 --> 30kb.
The computed matrix for the SRR1956527 dataset differs ~160 kb from the one computed with the master branch, both with a size of 29 MB. Maybe this is caused by different versions of the used compression or different handling of the used internal datatypes when the data is transfered to the sparse matrix etc?'